whylog/tests/tests_log_reader/test_files/003_match_time_range/expected_output.txt

```diff
@@ -1,5 +1,5 @@
 --- investigated item [node_1.log line 3]:
-3 visible effect
+2015-12-03 12:08:09 visible effect
```

Comment:
It was changed earlier in one of pull requests, I just made file `expected_output.txt` consistent with previous changes.